CHANGELOG

```diff
@@ -1,3 +1,10 @@
+Version 0.19.3 (2023-06-21)
```

Comment:
Should we also bump version in the README? I can see `"0.19.1"` there right now ;)

Reply:
right, doing it now